stacks/bracketstester.js

```diff
@@ -0,0 +1,30 @@
+const Stack = require('./Stack');
+
+function BracketsTester(stackMaxCapacity) {
+    this.stringStack = new Stack(stackMaxCapacity);
+    this.count = 0;
```

Comment:
why do you need count in here? and why do you even need to create a class for this problem, this could just be a function checkBrackets

Reply:
i am using count for counting the number of brackets. When a ")" is found, count is increased. and when a opening bracket "(" is found, count is decreased. So in the end, if brackets are balanced, count would remain 0.